chrome.js

```diff
@@ -77,12 +95,32 @@ class ChromeWebDriverInstaller extends WebDriverInstallerBase {
    * @return {!Promise}
    */
   async install(driverVersion, outputDirectory) {
+    const majorVersion = parseInt(driverVersion.split('.')[0], 10);
```

Comment:
nit; `parseInt(driverVersion.split('.')[0], 10)` can be extracted to a function.

Reply:
parseInt is already a function, right?